pages/price-feeds/use-real-time-data/solana.mdx

```diff
@@ -1,23 +1,149 @@
+console.log(priceUpdateData);
+```
+
+### Post price updates
+
+Finally, post the price update to the Pyth program on Solana.
+This step will create the price update account that your application reads from.
+Applications typically combine posting the price update and invoking their application into a sequence of transactions.
+The `PythSolanaReceiver` class in `@pythnetwork/pyth-solana-receiver` provides a convenient transaction builder to help with this process:
+
+```typescript copy
+import { PythSolanaReceiver } from "@pythnetwork/pyth-solana-receiver";
+
+// You will need a Connection from @solana/web3.js and an AnchorWallet to create
+// the receiver.
+const connection: Connection;
+const wallet: AnchorWallet;
+const pythSolanaReceiver = new PythSolanaReceiver({ connection, wallet });
+
+// Set closeUpdateAccounts: true if you want to delete the price update account at
+// the end of the transaction to reclaim rent.
+const transactionBuilder = pythSolanaReceiver.newTransactionBuilder({
+  closeUpdateAccounts: false,
+});
+await transactionBuilder.addPostPriceUpdates(priceUpdateData);
+
+// Use this function to add your application-specific instructions to the builder
+// await transactionBuilder.addPriceConsumerInstructions(...)
+
+await pythSolanaReceiver.provider.sendAll(
+  await transactionBuilder.buildVersionedTransactions({
+    computeUnitPriceMicroLamports: 50000,
+  }),
+  { skipPreflight: true }
+);
+```
+
+TODO: link to SDK documentation to show how to use this builder better.
```

Comment:
I can do this once the new docs for the SDK are published

Reply:
New sdk is out https://www.npmjs.com/package/@pythnetwork/pyth-solana-receiver